### ISSUE TITLE
Add game info about kickoff is counting down

### ISSUE THREAD

**Issue by NicEastvillage** [RLBot/RLBot#424]
This will allow boths to plan during kickoff countdown. This is especially useful for team play to decide who should go for kickoff.

**Comment by naturecodevoid** [comment]
I'm not very experienced, but I'm pretty sure the bots' `get_output()` don't run until the kickoff countdown is over. 

I'm not so sure about this either, but I think there is already a value in the packet, even though it seems not very useful to me.

**Comment by NicEastvillage** [comment]
That is not true. `get_output()` is still called during kickoffs, but just like players, their controls are mostly ignored. You can see this if you print something to the console every second.

You are thinking of the `is_kickoff_pause` but it is not what I have in mind. `is_kickoff_pause` is true during kickoff while the cars can move but the game timer is still paused. I would like a flag that is true during the kickoff countdown, the *3.. 2.. 1..*

This phase of the game is hard to detect and even harder than it used to be due to #423

**Comment by naturecodevoid** [comment]
Sorry, my bad! I have *no* idea how this would be added, but I could try with a little guidance. Again, sorry with the assumption!

**Comment by NicEastvillage** [comment]
Ideally we get the flag from the game, which means we need Psyonix to add it to the API. Or we fix #423 and botch it, but that requires access to the closed source.

**Comment by naturecodevoid** [comment]
I have no idea how you guys contact Psyonix, but if you've heard about Epic Games, they haven't said anything about still allowing things like RLBot, AlphaConsole, and BakkesMod (I don't think so). Also, Psyonix looks pretty busy with game updates and patches.
